.github/actions/next-stats-action/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt install unzip wget curl nano htop screen build-essential pkg-config libs
 
 RUN ln $(which python3) /usr/bin/python
 
-RUN curl -sfLS https://install-node.vercel.app/v18.17.0 | bash -s -- -f
+RUN curl -sfLS https://install-node.vercel.app/v21 | bash -s -- -f
```

Comment:
We could make this come from an env variable so it always installs the same version as defined in the workflow with `NODE_LTS_VERSION`

Reply:
this might be risky if we're using apis that available in new nodejs version but not supported in 18. we had similar problem before that we were using api that not supported in 16 but available in 17 or 18